cdn/dev/keyboard-search/search.js

```diff
@@ -145,12 +149,15 @@ function process_response(q, res) {
       $('.description', k).html(kbd.description);
 
       switch(kbd.match.type) {
-        case 'keyboard': $('.title a', k).mark(q); break; // don't annotate
-        case 'keyboard_id': $('.id', k).mark(q); break; // don't annotate
-        case 'language': $('.title .match', k).text('('+kbd.match.name+' language)').mark(q); break;
-        case 'region': $('.title .match', k).text('('+kbd.match.name+')').mark(q); break;
-        case 'script': $('.title .match', k).text('('+kbd.match.name+' script)').mark(q); break;
-        case 'description': $('.description', k).mark(q); break;
+        case 'keyboard': $('.title a', k).mark(qq); break; // don't annotate
+        case 'keyboard_id': $('.id', k).mark(qq); break; // don't annotate
+        case 'language': $('.title .match', k).text('('+kbd.match.name+' language)').mark(qq); break;
+        case 'language_id': $('.title .match', k).text('(language tag \''+kbd.match.name+'\')').mark(qq); break;
+        case 'country': $('.title .match', k).text('('+kbd.match.name+')').mark(qq); break;
+        case 'country_id': $('.title .match', k).text('(country id \''+kbd.match.name+'\')').mark(qq); break;
```

Comment:
I guess it is intentional that `region` is missing now?

Reply:
Yes, we were using only countries so moving to that naming scheme. You'll see another PR later which does some additional renaming.